app/javascript/packs/service.js

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue';
+import Vuelidate from 'vuelidate';
+import VueI18n from 'vue-i18n';
+import App from '../service/App.vue';
+import i18n from '../service/i18n';
+
+Vue.use(VueI18n);
+Vue.use(Vuelidate);
+
+const i18nConfig = new VueI18n({
+  locale: 'en',
+  messages: i18n,
+});
+
+// Event Bus
+window.bus = new Vue();
```

Comment:
The bus is not required in this pack, please remove it.

Reply:
It is required in `Branding` component https://github.com/chatwoot/chatwoot/pull/2685/files#diff-339c53cb115f488df2b3099dc110a67d36e3ab5a9b7611fc114a73dc385bf881